components/clarinet-files/src/network_manifest.rs

```diff
@@ -422,7 +433,7 @@ impl NetworkManifest {
                     .unwrap_or("devnet".to_string()),
                 bitcoin_controller_block_time: devnet_config
                     .bitcoin_controller_block_time
-                    .unwrap_or(30_000),
+                    .unwrap_or(90_000),
```

Comment:
Oh bummer, did it need to be slowed down to function properly?

Reply:
No, 30s still works as expected, but now that microblocks are correctly handled and we have visual feedback for it in the terminal UI, using a longer block time might be more faithful to what will happen on Mainnet.